medusa/providers/torrent/html/yggtorrent.py

```diff
@@ -0,0 +1,174 @@
+        """
+        results = []
+        if not self.login():
+            return results
+
+        # Search Params
+        search_params = {}
+
+        for mode in search_strings:
+            log.debug('Search mode: {0}', mode)
+
+            for search_string in search_strings[mode]:
+
+                if mode != 'RSS':
+                    log.debug('Search string: {search}',
+                              {'search': search_string})
+
+                search_params['q'] = re.sub(r'[()]', '', search_string)
+                response = self.session.get(self.urls['search'], params=search_params)
+                if not response or not response.text:
+                    log.debug('No data returned from provider')
+                    continue
+
+                results += self.parse(response.text, mode)
+
+        return results
+
+    def parse(self, data, mode):
+        """
+        Parse search results for items.
+
+        :param data: The raw response from a search
+        :param mode: The current mode used to search, e.g. RSS
+
+        :return: A list of items found
+        """
+        items = []
+
+        with BS4Parser(data, 'html5lib') as html:
+            torrent_table = html.find(class_='table table-striped')
+            torrent_rows = torrent_table('tr') if torrent_table else []
+
+            # Continue only if at least one Release is found
+            if len(torrent_rows) < 2:
+                log.debug('Data returned from provider does not contain any torrents')
+                return items
+
+            # Skip column headers
+            for result in torrent_rows[1:]:
+                cells = result('td')
+                if len(cells) < 5:
+                    continue
+                try:
+                    title = cells[0].find('a', class_='torrent-name').get_text(strip=True)
+                    download_url = urljoin(self.url, cells[0].find('a', target='_blank')['href'])
+                    if not (title and download_url):
+                        continue
+
+                    seeders = try_int(cells[4].get_text(strip=True))
+                    leechers = try_int(cells[5].get_text(strip=True))
+
+                    torrent_size = cells[3].get_text()
+                    size = convert_size(torrent_size, sep='') or -1
+
+                    # Filter unseeded torrent
+                    if seeders < min(self.minseed, 1):
+                        if mode != 'RSS':
+                            log.debug("Discarding torrent because it doesn't meet the"
+                                      " minimum seeders: {0}. Seeders: {1}",
+                                      title, seeders)
+                        continue
+
+                    item = {
+                        'title': title,
+                        'link': download_url,
+                        'size': size,
+                        'seeders': seeders,
+                        'leechers': leechers,
+                        'pubdate': None,
```

Comment:
Yeah date is in french, so I don't know if we should implement now. I think we should have some kind of translation table in place. @duramato do you know of another provider where we do pubdate translation?